DEVELOPMENT.md

```diff
@@ -256,6 +256,27 @@ The recommended minimum development configuration is:
   - 8 GB of (actual or virtualized) platform memory
 - Node autoscaling, up to 3 nodes
 
+#### Using [KinD](https://kind.sigs.k8s.io/)
+
+[KinD](https://kind.sigs.k8s.io/) is a great tool for working with Kubernetes clusters locally. It is particularly useful to quickly test code against different cluster [configurations](https://kind.sigs.k8s.io/docs/user/quick-start/#advanced).
+
+1. Install [required tools](./DEVELOPMENT.md#install-tools) (note: may require a newer version of Go).
+2. Install [Docker](https://www.docker.com/get-started).
```

Comment:
nit: or anything supported by `kind` (like `podman`)

Reply:
kind unfortunately doesn't have a good page to link to at the moment for supported runtimes, so I'm a bit hesitant to include this right now (I want to make the getting started instructions as simple as possible with minimal research needed).

Reply:
yeah make sense 👍🏼